attributes/network/9_account_features/1_account_multisig.adoc

```diff
@@ -51,14 +51,15 @@ This is left at the discretion of the server, but a value can be passed for each
 This feature adds 2 roles; `canMultisigSubmit` and `canMultisigApprove`.
 
 - `"canMultisigSubmit"` is allowed to submit new transactions, and to withdraw their own submitted transactions.
+    It also includes any permissions granted by `"canMultisigApprove"`.
 - `"canMultisigApprove"` is allowed to approve transactions, and revoke their own approvals.
 
 Additionally, `"owner"` is allowed to withdraw transactions, and update defaults.
```

Comment:
Nothing, and that might be a problem.

Reply:
Will do this.

Reply:
> If the sender is an account owner then it can change the defaults without others' approval.

This will be by design. I'll add a note that if you want you can set the account as its own owner. But the owner will always have all-powerful access to the account.